addons/sys_core/stringtable.xml

```diff
@@ -126,5 +126,25 @@
             <Japanese>ヘッドセットのトグル/スペクテイターのトグル</Japanese>
             <Italian>Attiva/Disattiva Cuffia - Attiva/Disattiva Spettatori</Italian>
         </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyInterference_displayName">
+            <English>Model Interference?</English>
```

Comment:
About deprecating the modules. I would be against it since most mission makers use them to quickly configure things, personalise the mission even further and bypass CBA settings.
I know the same can be done from init.sqf but, well, not everyone may do that. Just my 2 cents.